packages/litexa/src/parser/litexa.pegjs

```diff
@@ -370,6 +373,54 @@ DefineStatement "database type definition"
   }
 
 
+AllFileExclusions
+  = results:AllFileExclusionsStatment* {
+    if ( !results ) {
+      return true;
+    }
+    let shouldInclude = true;
+    for ( let i=0; i<results.length; ++i ) {
+      if ( !results[i] ) {
+        shouldInclude = false;
+      }
+    }
+    return shouldInclude;
+  }
+
+AllFileExclusionsStatment
+  = __ EndOfStatementSequence { return true; }
+  / r:ExcludeStatement __ EndOfStatementSequence { return r; }
+  / $(!EndOfStatementSequence .)* EndOfStatementSequence { return true; }
+
+/* litexa [exclude file]
+The `exclude file` statement lets you specify a static [expression](
+#expression) to exclude all the contents of the file. You might want to exclude
+a file of test states from a production build, or exclude a file from versions
+of your skill, as defined by your [DEPLOY](#deploy-variable) variables.
+
+```coffeescript
+exclude file unless DEPLOY.DEBUG
+# excludes a file's contents unless the DEPLOY variable DEBUG was set
+```
```

Comment:
Might be a little confusing since "tests" in the reader's understanding would be excluded regardless. Could perhaps reword along the lines of:

The `exclude file` statement can be used to skip a `.test.litexa` file if the tests within aren't relevant and would otherwise fail for the deployment target being tested. For instance:
```coffeescript
# campaign.test.litexa
exclude file unless DEPLOY.ENABLE_CAMPAIGN
# ... tests for the campaign mode that assume it's enabled
```